### PR TITLE
DB field rename: conv_mode -> mode

### PR DESCRIPTION
**Pull request by alexandraBara**
Renamed conv_mode to mode so it matches the Driver cmd fields.
This avoids having workarounds in the MITuna code to address the naming difference. 

Depends on: https://github.com/ROCmSoftwarePlatform/fin/pull/64

### CODE REVIEW

**Review by iq136boy — APPROVED** [review]

**Comment by cderb** [comment]
How does this address conv_mode in metadata / parsing / driver_conv?

**Comment by alexandraBara** [comment]
> How does this address conv_mode in metadata / parsing / driver_conv?

Those are all addressed in #783 
This PR only addressed the DB schema change through alembic.

File: alembic/versions/f3c8db790f9e_conv_mode_to_mode.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  op.alter_column('conv_config',
+                  'conv_mode',
+                  new_column_name='mode',
+                  existing_type=sa.VARCHAR(40),
+                  nullable=False,
+                  server_default="conv")
+
+
+def downgrade() -> None:
+  pass
```
Comment (JehandadKhan) [inline alembic/versions/f3c8db790f9e_conv_mode_to_mode.py]: Should we fill in the inverse of the above operation ?

**Review by JehandadKhan — CHANGES_REQUESTED** [review]

File: alembic/versions/f3c8db790f9e_conv_mode_to_mode.py (hunk 1 above)
Comment (alexandraBara) [reply alembic/versions/f3c8db790f9e_conv_mode_to_mode.py]: yes, ill update

**Comment by JehandadKhan** [comment]
@cderb  Please review
@alexandraBara Please remove the `OnHold` label when ready

**Review by JehandadKhan — APPROVED** [review]

**Pull request merged**